packages/@aws-cdk/aws-sns/lib/topic-ref.ts

```diff
@@ -168,21 +173,27 @@ export abstract class TopicRef extends cdk.Construct implements events.IEventRul
   /**
    * Adds a statement to the IAM resource policy associated with this topic.
    *
-   * If this topic was created in this stack (`new Topic`), a topic policy
-   * will be automatically created upon the first call to `addToPolicy`. If
-   * the topic is improted (`Topic.import`), then this is a no-op.
+   * If this topic was created in this stack (``new Topic`), a topic policy
+   * will be automatically created upon the first call to ``addToPolicy``. If
+   * the topic is improted (``Topic.import``), then this is a no-op.
```

Comment:
typo "improted"

Reply:
This one predates my edit (not my typo), hah!